filebeat/docs/multiple-prospectors.asciidoc

```diff
@@ -18,8 +23,12 @@ filebeat:
     -
       paths:
         - "/var/log/apache2/*"
+      fields:
+        json: true
+      fields_under_root: true
 -------------------------------------------------------------------------------------
 
-The config file in the example starts two prospectors. The first prospector has two harvesters,
+The config file in the example starts two prospectors (each prospector begins with a `-`). The first prospector has two harvesters,
```

Comment:
@urso OK, I'll make this sound less magical. :-) But I think the change for input_type to come after the dash is for master, right, and not 1.1? If so, I'll change the example after I've cherry-picked the commit into master.